src/refactor_nrepl/ns/tracker.clj

```diff
@@ -4,7 +4,7 @@
             [clojure.tools.namespace
              [dependency :as dep]
              [file :as file]
-             [repl :refer [refresh-dirs]]
+             [repl :as tools.namespace.repl]
```

Comment:
That's inconsistent with the rest of the aliases. I kind of understand the reasoning, but I don't like inconsistencies, especially without explanations. :-)

Reply:
`repl` in particular has bit of an unfortunate name, in principle the `refresh-dirs` aren't exclusively concerned with the repl.

When using refactor-nrepl one might not interact with a repl at all to get certain stuff done.

The `tools.namespace.repl` alias moves weight off 'repl' and into 'tools.namespace'.

Reply:
Yeah, I got this. I meant it probably won't be clear for everyone. I'd just adopt consistent aliases like `c.t.n.repl`, `c.t.n.file`, etc. Or something along those lines at least.

Reply:
Hmmm, I like / I'm used to `file` as an alias from various projects. 

Maybe I can set up `:consistent-alias` in clj-kondo for enforcing that we're consistently inconsistent :) i.e. just one exception to these short names.

I'd add an explanatory comment as well to that linting config. SGTY?

Reply:
> I'd add an explanatory comment as well to that linting config

(done, tentatively)